components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/EntitlementNotificationExtension.java

```diff
@@ -36,7 +37,7 @@
  * registered in entitlement.properties file.
  */
 @SuppressWarnings("unused")
-public class EntitlementNotificationExtension implements PAPStatusDataHandler {
```

Comment:
This change wont be needed as we discussed to keep the `PAPStatusDataHandler` untouched.